### PR TITLE
Normative: Revert U+2212 from time zone offset grammar

### PR DESCRIPTION
**Pull request by justingrant**
This PR is the Temporal counterpart to https://github.com/tc39/ecma262/pull/3334.

See that PR for details.

### CODE REVIEW

File: spec/mainadditions.html [hunk 1]
```diff
@@ -514,9 +514,9 @@ <h1>
         1. Let _parseResult_ be ParseText(StringToCodePoints(_offsetString_), <del>|UTCOffset|</del><ins>|UTCOffsetSubMinutePrecision|</ins>).
         1. <del>Assert: _parseResult_ is not a List of errors.</del>
         1. <ins>If _parseResult_ is a List of errors, throw a *RangeError* exception.</ins>
-        1. Assert: _parseResult_ contains a |TemporalSign| Parse Node.
-        1. Let _parsedSign_ be the source text matched by the |TemporalSign| Parse Node contained within _parseResult_.
-        1. If _parsedSign_ is the single code point U+002D (HYPHEN-MINUS) or U+2212 (MINUS SIGN), then
+        1. Assert: _parseResult_ contains a |ASCIISign| Parse Node.
+        1. Let _parsedSign_ be the source text matched by the |ASCIISign| Parse Node contained within _parseResult_.
+        1. If _parsedSign_ is the single code point U+002D (HYPHEN-MINUS), then
```
Comment (ptomato) [inline spec/mainadditions.html:519]: These would be included in the ECMA-262 PR, yes?

**Review by ptomato — APPROVED** [review]

File: spec/mainadditions.html (hunk 1 above)
Comment (justingrant) [reply spec/mainadditions.html:519]: If you're asking whether https://github.com/tc39/ecma262/pull/3334 also includes replacing "TemporalSign" with "ASCIISign", then yes it does.

**Comment by ptomato** [comment]
This normative change reached consensus at the TC39 meeting of 2024-06-12. It doesn't need to be merged exactly at the same time as ecma262#3334, however, we should write test262 tests before merging either of them.

**Comment by ptomato** [comment]
Disregard the codecov message - this still needs an implementation. I'll work on the test262 tests and then use that to develop an implementation which I'll tack on to this PR.

**Comment by ptomato** [comment]
Tests are in https://github.com/tc39/test262/pull/4120 and I've added the corresponding implementation to this PR.

**Pull request merged**